operators/cp4ba-operator/overlays/latest/kustomization.yaml

```diff
@@ -0,0 +1,81 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+
+namespace: cp4ba
+
+bases:
+  - ../../base
+
+patches:
+  - target:
+      kind: Subscription
+      name: ibm-cp4a-operator-catalog-subscription
+    patch: |-
+      - op: replace
+        path: /spec/channel
+        value: 'v22.1'
+
+---
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
+
+namespace: openshift-marketplace
+
+bases:
+  - ../../base
+
+patches:
```

Comment:
You can add these entries to the above patches stanza defined in line 9. They don't all have to be the same target.kind spec